vcr/api/v1/api_test.go

```diff
@@ -265,21 +266,7 @@ func TestWrapper_Search(t *testing.T) {
 		}
 
 		assert.Len(t, capturedConcept, 1)
-		assert.Equal(t, "did:nuts:B8PUHs2AUHbFF1xLLK4eZjgErEcMXHxs68FteY7NDtCY#123", capturedConcept[0]["id"])
-		assert.Equal(t, "HumanCredential", capturedConcept[0]["type"])
-	})
-
-	t.Run("error - unknown template", func(t *testing.T) {
```

Comment:
with the movement of code from API to vcr, this test has been removed for the API. It has not been added again on the vcs_test.